blueprints/helper-addon/files/__root__/__path__/__name__.ts

```diff
@@ -0,0 +1 @@
+export { default, <%= camelizedModuleName %> } from '<%= modulePath %>';
```

Comment:
@dfreeman is this kind of re-export going to work/compile properly as TypeScript? I *think* the answer we got to was *yes, everything works except things in `app/` in the root of an addon* but wanted to confirm.

Reply:
I think we want these to just be `.js` files in the app tree — I can open a PR to update the blueprints we have in place